### PR TITLE
xds: rollback PGV dependency from using maven artifact to importing proto source

### PR DESCRIPTION
**Pull request by dapengzhang0**
PGV is adding new fields (`well_known_regex` and `strict`) to proto and some of our new features require the most current envoy API which in turn requires the new fields in latest PGV proto (https://github.com/envoyproxy/envoy/blob/master/api/envoy/api/v2/core/base.proto#L251), but they are not available in the latest PGV maven release.

### CODE REVIEW

File: xds/build.gradle [hunk 1]
```diff
@@ -77,7 +74,6 @@ shadowJar {
     classifier = null
     dependencies {
         include(project(':grpc-xds'))
-        include(dependency('io.envoyproxy.protoc-gen-validate:'))
     }
     relocate 'com.github.udpa', 'io.grpc.xds.shaded.com.github.udpa'
     relocate 'io.envoyproxy', 'io.grpc.xds.shaded.io.envoyproxy'
```
Comment (ejona86) [inline xds/build.gradle]: Seems like we would need a new relocation.

Comment (dapengzhang0) [reply xds/build.gradle]: Are you referring to shadowing PGV?
https://github.com/grpc/grpc-java/blob/ad5701815c6659968a5b47fb3c3cbe17bf5d3ed8/xds/third_party/protoc-gen-validate/src/main/proto/validate/validate.proto#L5

The parent package `io.envoyproxy` is relocated.

Comment (ejona86) [reply xds/build.gradle]: Yeah. Okay. Sounds good.

File: xds/third_party/protoc-gen-validate/import.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Copyright 2018 The gRPC Authors
```
Comment (sanjaypujare) [inline xds/third_party/protoc-gen-validate/import.sh]: should change year to 2020?

Comment (dapengzhang0) [reply xds/third_party/protoc-gen-validate/import.sh]: Good question. I don't know the answer, will keep it as is.

**Pull request merged**